### PR TITLE
Metrics api events

### LINKED ISSUE #1127: [Event] Save-as workflow for "Metric" type
Event explorer needs "Metric" type for Save-as.
Store same data as visualization, with addition of label and metric-scale/units

### PR DESCRIPTION
**Pull request by kavithacm**
### Description
Save button API workflow under event analytics for metrics

### Issues Resolved
#1127 #1129 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: dashboards-observability/public/components/event_analytics/explorer/save_panel/save_panel.tsx [hunk 1]
```diff
@@ -149,11 +162,11 @@ export const SavePanel = ({
...
                 />
               </EuiFormRow>
-              <EuiSpacer size="s" />
+              {/* <EuiSpacer size="s" />
               <EuiTitle size="xxs">
                 <h3>{'Labels'}</h3>
-              </EuiTitle>
-              <EuiFormRow>
+              </EuiTitle> */}
+              {/* <EuiFormRow>
```
Comment (sejli) [inline dashboards-observability/public/components/event_analytics/explorer/save_panel/save_panel.tsx:169]: remove comments if unnecessary? unless we are still deciding on using labels

**Review by sejli — APPROVED** [review]
looks good to me!

File: dashboards-observability/public/components/event_analytics/explorer/save_panel/save_panel.tsx (hunk 1 above)
Comment (kavithacm) [reply dashboards-observability/public/components/event_analytics/explorer/save_panel/save_panel.tsx:169]: let this component be commented for now(can be used later), need to finish some more work in events home page

**Review by pjfitzgibbons — APPROVED** [review]

**Comment by kavithacm** [comment]
> 

those are just failed tests, thats fine for now

**Pull request merged**